client/login/wp-login/style.scss

```diff
@@ -366,5 +371,59 @@ $image-height: 47px;
 		// change border color to remove 3D effect
 		background-color: #007cba;
 		border-color: #007cba;
+		color: #fff;
+		width: auto;
+		min-width: 170px;
+		text-align: center;
+		font-size: 14px;
+		line-height: 14px;
+		height: 42px;
+		display: block;
+		margin: 0 auto;
+		float: none;
+		font-weight: 500;
+		letter-spacing: 0;
+		transition: background 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.2s ease-in-out;
+		border-radius: 4px;
+		box-shadow: none;
+
+		&:hover,
+		&:active {
+			background: #72aee6;
+			border-color: #72aee6;
+			color: #fff;
+			outline-color: transparent;
+			box-shadow: none;
+		}
+	}
+
+	.two-factor-authentication__verification-code-form > p,
+	.login__form .login__form-userdata label,
+	.login__form-terms,
+	.login__social-tos {
+		color: var( --studio-gray-40 );
+	}
+
+	.login__form-terms a,
+	.login__social-tos a {
+		color: var( --studio-gray-80 );
+	}
+
+	.two-factor-authentication__verification-code-form input,
+	.login__form input {
+		padding: 18px 14px;
+		font-size: 14px;
+	}
+
+	.wp-login__main {
+		max-width: 540px;
```

Comment:
The container is good and wide now, but it means the form fields are a bit wider than in the signup form. Would it work to update this selector to something like:

```
.wp-login__main {
  max-width: 540px;

  form {
    max-width: 440px;
    margin: 0 auto;
  }
}
```

Looks like the classes on the login page are a little awkward for this (because `.login__form` doesn't include the social signup buttons)